app/views/devise/sessions/new.html.erb

```diff
@@ -1,4 +1,5 @@
-<h2>Log in</h2>
+<div id="container">
+<h2 id="log-in">Log in</h2>
```

Comment:
This should be indented 2 space.

Reply:
Also, actually, looks like everything inside of the div isn't indented. Any time an element is nested in another element, it should be indented 2 lines form the parent element's indentation.